### PR TITLE
[Supermicro]: Add a new supported device and platform, SSE-T7132S.

### PR DESCRIPTION
**Pull request by wilson-smci**
#### Why I did it
* platform/innoviunm: Add a new supported device and platform, SSE-T7132S

* Switch Vendor: Supermicro
* Switch SKU:  Supermicro_sse_t7132s
* ASIC Vendor: innovium
* Swich ASIC: TL7
* Port Configuration: 32x400G

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
1. Add the files into device/supermicro/x86_64-supermicro_sse_t7132s-r0/ and platform/innovium/sonic-platform-modules-supermicro/ for SSE-T7132S.
2. Modify the make files in platform/innovium/ for SSE-T7132S.
#### How to verify it
1. Build a Innovium target for SSE-T7132S.

2. Run this image on SSE-T7132S and verify all dockers are up and test basic commands like:
show version
show platform summary
show platform syseeprom
show platform fan
show platform psustatus
show platform firmware status
show platform temperature
show platform ssdhealth

3. Verify ports are up using "show interface status" command

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Comment by jiachi-jau** [comment]
@lguohan Please review this PR, Thanks

**Comment by puffc** [comment]
@prgeor @lguohan  Could you please help review and merge this PR? Thanks.

File: device/supermicro/x86_64-supermicro_sse_t7132s-r0/platform_reboot
Comment (prgeor) [inline device/supermicro/x86_64-supermicro_sse_t7132s-r0/platform_reboot:1]: @wilson-smci can this file be removed? its empty

Comment (wilson-smci) [reply device/supermicro/x86_64-supermicro_sse_t7132s-r0/platform_reboot:1]: Hi I remove the dummy file. 
Please review it.
Thanks.

**Comment by puffc** [comment]
@prgeor If you have more comments please let us know. Thanks!

**Review by prgeor — APPROVED** [review]
@lguohan @yxieca please help merge/review

**Comment by wilson-smci** [comment]
@lguohan @yxieca please help merge/review
Thanks

**Comment by puffc** [comment]
@yxieca Many thanks! BTW, I'm wondering if you can add Request for 202111 branch and Request for 202211 branch lables for this PR?

**Comment by wilson-smci** [comment]
> @yxieca Many thanks! BTW, I'm wondering if you can add Request for 202111 branch and Request for 202211 branch lables for this PR?

@yxieca Please help us to merge this feature into 202111 branch.

**Comment by yxieca** [comment]
> > @yxieca Many thanks! BTW, I'm wondering if you can add Request for 202111 branch and Request for 202211 branch lables for this PR?
> 
> @yxieca Please help us to merge this feature into 202111 branch.

@wilson-smci in general, we don't cherry-pick feature back into feature branches.

**Comment by puffc** [comment]
> > > @yxieca Many thanks! BTW, I'm wondering if you can add Request for 202111 branch and Request for 202211 branch lables for this PR?
> > 
> > 
> > @yxieca Please help us to merge this feature into 202111 branch.
> 
> @wilson-smci in general, we don't cherry-pick feature back into feature branches.

@yxieca It make sense to me. Thank you for the clarification.

**Pull request merged**